src/main/scala/optics/internal/focus/ErrorHandling.scala

```diff
@@ -7,7 +7,7 @@ private[focus] trait ErrorHandling {
     case FocusError.NotACaseClass(fromClass) => s"Expecting a case class in the 'From' position; found $fromClass"
     case FocusError.NotAConcreteClass(fromClass) => s"Expecting a concrete case class in the 'From' position; cannot reify type $fromClass"
     case FocusError.NotASimpleLambdaFunction => s"Expecting a lambda function that directly accesses a field. Example: `GenLens[Address](_.streetNumber)`"
-    case FocusError.DidNotDirectlyAccessArgument => s"Expecting a lambda function that directly accesses a field. Example: `GenLens[Address](_.streetNumber)`"
+    case FocusError.DidNotDirectlyAccessArgument(argName) => s"Expecting a lambda function that directly accesses the argument; other variable `$argName` found. Example: `GenLens[Address](_.streetNumber)`"
```

Comment:
should it be `Focus[Address](_.streetNumber)` or `address.focus(_.streetNumber)`